data/human/south jobs.txt

```diff
@@ -15,7 +15,8 @@ mission "Pirate Occupation [0]"
 	name `Pirates occupying <system>`
 	minor
 	repeat
-	description `A small band of pirates have been occupying <system>, and the local government has asked for help. Defeat the pirates in the system and land on <planet> when done. Payment is <payment>.`
+	description `A small band of pirates have been occupying <system>, and the local government has asked for help. Defeat the pirates in the system and land on <planet> before <deadline>, to receive the payment of <payment>.`
```

Comment:
If the deadline is the day after shouldnt we mention it that way?
Also none of these descriptions are too long

Reply:
No, sorry, I haven't checked the length. I'll get to that soon.

Can you explain what you're asking with the deadline with a suggested change? I'm not sure what you mean.

Reply:
It's not too long, but the substitutions don't work. (I changed it on my save to <date> but it made no difference.)
<img width="1680" alt="Screen Shot 2023-07-01 at 7 15 59 AM" src="https://github.com/endless-sky/endless-sky/assets/125756761/b3e93bda-7d48-4ff0-9a18-a305fac2a7cf">

Reply:
```suggestion
	description `A small band of pirates have been occupying <system>, and the local government has asked for help. Defeat the pirates in the system and land on <planet> in the same day, to receive the payment of <payment>.`
```
I meant smth like this
Also if it doesnt work its probably because of the wrong substitution being used?

Reply:
The `before <deadline>` refers to both landing and when you get there.
And I've used `<deadline>` and `<date>` and no substitutions.